### PR TITLE
Created block program

### PR DESCRIPTION
**Pull request by Flasto4ka**
image

### CODE REVIEW

**Comment by flast1k** [comment]
Need to fix a few issues:
image
image
image

File: css/style.css [hunk 1]
```diff
@@ -469,3 +471,94 @@ ul {
...
 }
+
+/* Block program */
+
+.program {
+  background-color: #eef1f2;
+}
+
+.program__text {
+  position: absolute;
```
Comment (flast1k) [inline css/style.css:482]: position: relative

File: index.html [hunk 2]
```diff
@@ -383,6 +383,58 @@ <h1 class="banner__title">
...
+      <!-- Block program -->
+
+      <section class="program">
+        <div class="container">
+          <div class="row">
+            <div class="column-lg-9">
+              <div class="program__text">
+                <div class="program__title">Программа курса</div>
+
+                <div class="program__square-front">
```
Comment (flast1k) [inline index.html:396]: check these articles to implement accordion
https://doka.guide/html/details/
https://nikitahl.com/native-html-accordion

File: css/style.css (hunk 1 above)
Comment (Flasto4ka) [reply css/style.css]: +

File: css/style.css [hunk 3]
```diff
@@ -469,3 +471,94 @@ ul {
...
+  font-size: 24px;
+  line-height: 32px;
+  font-weight: bold;
+  margin-top: 150px;
+  margin-bottom: 60px;
+}
+
+.program__square-front {
+  width: 867px;
+  height: 188px;
```
[the remark this answers is not in the document]
Comment (Flasto4ka) [reply css/style.css]: +

File: index.html (hunk 2 above)
Comment (Flasto4ka) [reply index.html]: +

**Review by flast1k — APPROVED** [review]

**Pull request merged**